code/player/Player.Camera.cs

```diff
@@ -4,6 +4,17 @@ namespace Cinema;
 
 public partial class Player
 {
+    // default
+    public bool isRightShoulderView;
+
+    public void ThirdPersonSwapShoulder()
+    {
+    if (!ThirdPersonCamera)
```

Comment:
Not indented properly, use an automatic code styler like CSharpier.

Reply:
Fixed